Polyhedron/demo/Polyhedron/Plugins/Operations_on_polyhedra/Clip_polyhedron_plugin.cpp

```diff
@@ -117,6 +127,26 @@ public :
     connect(actionStartClippingWithPoly, &QAction::triggered,
             this, &Clip_polyhedron_plugin::start_clipping_with_poly);
 
+    connect(ui_widget.helpButton, &QPushButton::clicked,
+            [this](){
+      if(is_plane)
+        QMessageBox::information(dock_widget, QString("Help"),
+                                 QString("Clipping\n"
+                                         "\n"
+                                         "This function allows to clip all the selected polyhedra against a halfspace. "
+                                         "What is on the blue side of the clipping plane will be clipped, and what is on the yellow side will be kept.\n\n"
+                                         "If the option `keep closed` is checked, the clipped part of each polyhedron will be closed, "
+                                         "if it has a closed contour on the clipping plane. Otherwise, it will be left open."));
+      else
+        QMessageBox::information(dock_widget, QString("Help"),
+                                 QString("Clipping\n"
+                                         "\n"
+                                         "This function allows to clip all the selected polyhedra against a closed polyhedron, bounding a volume. "
+                                         "What is on the outside of the bound volume will be clipped, and what is on the inside will be kept.\n\n"
+                                         "If the option `keep closed` is checked, the clipped part of each polyhedron will be closed, "
+                                         "if it has a closed contour on the clipping polyhedron. Otherwise, it will be left open."));
```

Comment:
That is the sort of code where the usage of [C++11 raw string literals](https://abseil.io/tips/64) might help the readability.